src/migrateCalculatedMeasuresToActivePivot/removeCalculatedMemberDefinitionFromMDXAndGetCubeName.test.ts

```diff
@@ -0,0 +1,85 @@
+import { dataModelsForTests } from "@activeviam/data-model";
+import { MdxString } from "@activeviam/mdx";
+import { removeCalculatedMemberDefinitionFromMDXAndGetCubeName } from "./removeCalculatedMemberDefinitionFromMDXAndGetCubeName";
+
+const mdxStringWithNoCalculatedMeasures: MdxString =
+  "SELECT NON EMPTY Hierarchize(Descendants({[Geography].[City].[AllMember]}, 1, SELF_AND_BEFORE)) ON ROWS, NON EMPTY {[Measures].[contributors.COUNT]} ON COLUMNS FROM [EquityDerivativesCube] CELL PROPERTIES VALUE, FORMATTED_VALUE, BACK_COLOR, FORE_COLOR, FONT_FLAGS";
+
+const mdxStringWithOneCalculatedMeasure: MdxString =
+  'WITH  Member [Measures].[Distinct count city] AS Count(Descendants([Geography].[City].CurrentMember, [Geography].[City].[City]), EXCLUDEEMPTY), FORMAT_STRING = "#,###.##"  SELECT NON EMPTY Hierarchize(Descendants({[Currency].[Currency].[AllMember]}, 1, SELF_AND_BEFORE)) ON ROWS, NON EMPTY {[Measures].[contributors.COUNT], [Measures].[Distinct count city]} ON COLUMNS FROM [EquityDerivativesCube] CELL PROPERTIES VALUE, FORMATTED_VALUE, BACK_COLOR, FORE_COLOR, FONT_FLAGS';
+
+const mdxStringWithTwoCalculatedMeasures: MdxString =
+  'WITH  Member [Measures].[Log pv.SUM] AS Log([Measures].[pv.SUM], 10), FORMAT_STRING = "#,###.##"    Member [Measures].[Distinct count city] AS Count(Descendants([Geography].[City].CurrentMember, [Geography].[City].[City]), EXCLUDEEMPTY), FORMAT_STRING = "#,###.##"  SELECT NON EMPTY Hierarchize(Descendants({[Geography].[City].[AllMember]}, 1, SELF_AND_BEFORE)) ON ROWS, NON EMPTY {[Measures].[Log pv.SUM], [Measures].[Distinct count city]} ON COLUMNS FROM [EquityDerivativesCube] CELL PROPERTIES VALUE, FORMATTED_VALUE, BACK_COLOR, FORE_COLOR, FONT_FLAGS';
+
+const namesOfCalculatedMeasurestoMigrate = [
+  "Distinct count city",
+  "Exp gamma sum",
+  "Log pv.SUM",
+  "activeui5 calculated measure",
+  "Test calculated measure",
+  "pvSum ^ 2",
+];
+
+const dataModel = dataModelsForTests.sandbox;
+
+describe("removeCalculatedMemberDefinitionFromMDXAndGetCubeName", () => {
+  it("returns `calculatedMeasuresWithCubeNames` as an empty object and does not modify the MDX when there are no calculated measures", () => {
+    const removeCalculatedMeasures =
```

Comment:
Lets not use a verb-like name for a variable that is not a function.
You could just destructure the function output:
`const {cubeName, namesOfMigratedCalculatedMeasures, migratedMdx} = migrateCalculatedMeasuresInMdx(...)`.

Same for all tests.